classes/04class/exercises/c04-iac01/nimmi89/ec2.tf

```diff
@@ -0,0 +1,26 @@
+# Create a EC2 instance
+resource "aws_instance" "da-instance" {
+  ami = "ami-088ff0e3bde7b3fdf"
+  instance_type = "t2.micro"
+  vpc_security_group_ids = [aws_security_group.DA-sg.id]
+  key_name = "C01-AWS01KeyPair"
+ 
+  tags = {
+     Name = "DA-Instance-Terraform"
+ }
+ 
+ user_data = <<-EOF
+              #!/bin/bash
+              yum update -y
+              yum install -y httpd
+              curl 169.254.169.254/latest/meta-data/hostname > index.html
+              EOF
```

Comment:
Any particular reason to add this script here?

Reply:
Oh! I was trying how to pass user data through terraform for my reference. Forgot to take it off! I Have removed those lines!Thanks!

Reply:
No worries, nice that you are playing around with the options 👏👏

Now you can redo all the previous exercises using terraform 🤣🤣🤣